Software/Development/SemesterProject/App/controlTask.c

```diff
@@ -25,8 +25,8 @@ void executeControl(void)
          else
          {
             // Call closed-loop function
-//             runStartUpControl();
-             runClosedLoopControl();
+             runStartUpControl();
```

Comment:
startup control must be updated to remove V/f stuff and add a proper increment of dutys and PI parameters